api/scripts/bigint/answer/add-triggers-to-copy-data-from-answer-ke-to-tmp-bigint-tables.js

```diff
@@ -0,0 +1,116 @@
+require('dotenv').config();
```

Comment:
**nitpick**
Exprimer l'intention en renommant
`add-triggers-to-copy-data-from-answer-ke-to-tmp-bigint-tables.js`
en
`copy-newly-inserted-rows-to-answerId-tmp-bigint-tables.js`